src/global/project/manifest.rs

```diff
@@ -357,11 +357,11 @@ impl Manifest {
     }
 
     /// Removes exposed mapping from the manifest
-    pub fn remove_exposed_name(
+    pub fn remove_exposed_name<'a>(
         &mut self,
         env_name: &EnvironmentName,
-        exposed_name: &ExposedName,
-    ) -> miette::Result<()> {
+        exposed_name: &'a ExposedName,
+    ) -> miette::Result<&'a ExposedName> {
```

Comment:
I don't get that one. Why would you need to return the same exposed_name that you passed yourself to the function?

Reply:
removed it